core/src/main/java/greencity/controller/ManagementOrderController.java

```diff
@@ -139,29 +135,6 @@ public ResponseEntity<HttpStatus> addCertificate(
         return new ResponseEntity<>(HttpStatus.CREATED);
     }
 
-    /**
-     * Controller delete certificate.
-     *
-     * @param code {@link String}.
-     * @return {@link HttpStatus} - http status.
-     * @author Hlazova Nataliia
-     */
-
-    @ApiOperation("Delete Certificate")
-    @ApiResponses(value = {
-        @ApiResponse(code = 200, message = HttpStatuses.OK),
-        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
-        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
-        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN)
-    })
-    @PreAuthorize("@preAuthorizer.hasAuthority('EDIT_CERTIFICATE', authentication)")
-    @DeleteMapping("/deleteCertificate/{code}")
-    public ResponseEntity<HttpStatus> deleteCertificate(
```

Comment:
I see that we can add `certificates`
Will we never delete them?

Reply:
This is not in the requirements for the project. But this possibility should be left. You are right, I should leave the endpoint to remove the Certificate.